qml/components-Sailfish/IconButton.qml

```diff
@@ -0,0 +1,30 @@
+import Sailfish.Silica 1.0
+import "." as BVApp
+
+IconButton {
+    function iconBy(type) {
```

Comment:
Thank. I think it's a good idea to move it into the Themes, but shall the function then return an array with `[iconString, scalingFactor]` or what did you think of?